src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -1534,7 +1534,7 @@ def test_builtin_single_OpenMP_pdo(monkeypatch, annexed):
     for dist_mem in [False, True]:
         _, info = parse(os.path.join(BASE_PATH,
                                      "15.7.2_setval_X_builtin.f90"),
-                        api=TEST_API, distributed_memory=dist_mem)
+                        api=TEST_API)
```

Comment:
Please could you update all of the tests you've touched to use the `dist_mem` fixture.

Reply:
I presume you mean the year.

Reply:
I've changed them all rather than just the ones I touched. I thought this would be just as easy (but I was wrong - there were lots to change!).